packages/@ember/-internals/runtime/lib/mixins/promise_proxy.d.ts

```diff
@@ -0,0 +1,24 @@
+import { Mixin } from '@ember/-internals/metal';
+
+interface PromiseProxyMixin<T> {
+  reason: null;
```

Comment:
The default value is presumably `null` but this would make it so you can't use anything *but* `null`. From the docs, I think it should be `unknown`?

```suggestion
  reason: unknown;
```